### ISSUE TITLE
The extension thumbnail should have an empty ALT

### ISSUE THREAD

**Issue by neeleshb** [microsoft/vscode#190434]
Type: <b>Bug</b>

WCAG 2.0 recommends that every  tag must have an ALT attribute. The value should be empty if the image is purely decorative.
https://www.w3.org/TR/WCAG20-TECHS/H37.html

Extension thumbnails are decorative, so they should have empty ALT.

VS Code version: Code 1.78.0 (252e5463d60e63238250799aef7375787f68b4ee, 2023-05-03T20:09:46.685Z)
OS version: Linux x64 6.3.11-1rodete1-amd64
Modes:
Sandboxed: No

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.81.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by sandy081** [comment]
Can you please mention the reason behind the empty alt text?

**Comment by neeleshb** [comment]
It's added in the original bug description. Repeating her for convenience - 

WCAG 2.0 recommends that every tag must have an ALT attribute. The value should be empty if the image is purely decorative.  https://www.w3.org/TR/WCAG20-TECHS/H37.html

**Comment by sandy081** [comment]
I read your description, but I could not find it in the link you provided (I might missed it). Can you please point me to the exact section where it is mentioned. Thanks.

**Comment by neeleshb** [comment]
Oh the exact wording isn't there on that link. But it points to the [HTML 4.01 alt attribute](http://www.w3.org/TR/html401/struct/objects.html#adef-alt) which says the following:

`The alt attribute must be specified for the IMG and AREA elements... `

`...In such cases, the alternate text should be the empty string ("")`

The general idea is that the alt text helps the screen readers to understand that the image is decorative. If there is no alt attribute, then the Screen readers will try to read the file name. Here are a few links that explain this, with the appropriate text quoted: 

1) https://a11y-101.com/development/the-alt-attribute - "Screen readers can’t read the alternative because there isn’t one. Instead the screen reader will read the file name."
2) https://accessibility.huit.harvard.edu/describe-content-images - "Leave alt text blank if the image is purely descriptive"
3) https://accessibility.asu.edu/articles/images - "all images must have alternative text that describes them, except for decorative images, which should have an empty ALT tag."
4)  https://accessibility.psu.edu/images/imageshtml/#empty - "With no ALT tag,  screen reader says 'Image' which leaves the user wondering if they are missing anything important

**Comment by sandy081** [comment]
Thanks for the explanation.
